web/src/features/fireWatch/components/FireWatchDashboard.tsx

```diff
@@ -1,18 +1,226 @@
-import { Box, Typography, useTheme } from "@mui/material"
+import { selectBurnForecasts } from '@/app/rootReducer'
+import { AppDispatch } from '@/app/store'
+import DetailPanelContent from '@/features/fireWatch/components/DetailPanelContent'
+import FireWatchDetailsModal from '@/features/fireWatch/components/FireWatchDetailsModal'
+import {
+  BurnForecast,
+  BurnStatusEnum,
+  FireWatch,
+  FireWatchBurnForecast,
+  FuelTypeEnum,
+  PrescriptionEnum
+} from '@/features/fireWatch/interfaces'
+import { fetchBurnForecasts } from '@/features/fireWatch/slices/burnForecastSlice'
+import InfoIcon from '@mui/icons-material/Info'
+import { Box, styled, Typography, useTheme } from '@mui/material'
+import {
+  DataGridPro,
+  DataGridProProps,
+  GridActionsCellItem,
+  GridColDef,
+  GridValueFormatterParams
+} from '@mui/x-data-grid-pro'
+import { isNull } from 'lodash'
+import { DateTime } from 'luxon'
+import React, { useEffect, useState } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
 
+export interface BurnWatchRow {
+  id: number
+  title: string
+  fireCentre: string
+  station: string
+  fuelType: FuelTypeEnum
+  status: BurnStatusEnum
+  burnWindowStart: DateTime
+  burnWindowEnd: DateTime
+  inPrescription: PrescriptionEnum
+  fireWatch: FireWatch
+  burnForecasts: BurnForecast[]
+}
+
+const StyledDataGrid = styled(DataGridPro)(({ theme }) => ({
+  ['&.in-prescription-yes']: { backGroundColor: theme.palette.success },
+  ['&.in-prescription-hfi']: { backGroundColor: theme.palette.warning }
+}))
 
 const FireWatchDashboard = () => {
+  const dispatch: AppDispatch = useDispatch()
+  const burnForecasts = useSelector(selectBurnForecasts)
   const theme = useTheme()
+  const [rows, setRows] = useState<BurnWatchRow[]>([])
+  const [modalOpen, setModalOpen] = useState(false)
+  const [selectedFireWatch, setSelectedFireWatch] = useState<FireWatchBurnForecast | null>(null)
+
+  const getInPrescription = (burnForecasts: BurnForecast[]): PrescriptionEnum => {
+    let inPrescription = PrescriptionEnum.NO
+    for (const burnForecast of burnForecasts) {
+      if (burnForecast.inPrescription === PrescriptionEnum.HFI) {
+        inPrescription = PrescriptionEnum.HFI
+      }
+      if (burnForecast.inPrescription === PrescriptionEnum.ALL) {
+        inPrescription = PrescriptionEnum.ALL
+        break
+      }
+    }
+    return inPrescription
+  }
+
+  const shapeRows = (values: FireWatchBurnForecast[]) => {
```

Comment:
This can be put in the `selectBurnForecasts` if it always has to shape rows this way.

Reply:
Good call! Done in https://github.com/bcgov/wps/pull/4493/commits/1aa13ef419d718b88e38ac9e72ae157e476cb1ee

Reply:
We can get away with it because we aren't doing any editing in the datagrid. If we eventually allow editing we may have to switch back to having the rows in local state.